indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -79,22 +105,33 @@
                 success: function(data) {
                     var $page = $('.management-page');
                     if (data.page_html) {
-                        $page.replaceWith(data.page_html);
-                        $page = $('.management-page');
+                        var $newPage = $(data.page_html);
+                        $page.replaceWith($newPage);
+                        $page = $newPage;
                     } else {
                         $box.replaceWith(data.box_html);
                     }
                     initForms($page);
                     showFormErrors($page);
                 }
             });
-        }).on('click', '.change-review-button', function(evt) {
+        }).on('click', '.js-change-review', function(evt) {
             evt.preventDefault();
 
             var $reviewDisplay = $(this).closest('.i-timeline-item');
             var $reviewBox = $reviewDisplay.next('.abstract-review-box');
             $reviewDisplay.hide();
             $reviewBox.show();
+        }).on('declarative:success', '.js-delete-comment', function() {
+            $(this).parents('.i-timeline-item').remove();
+        }).on('declarative:success', '.js-edit-comment', function(evt, data) {
+            var $oldContent = $(this).parents('.i-timeline-item').find('.i-box-content');
+            var $newContent = $(data.form_html);
+            $oldContent.replaceWith($newContent);
+            $newContent.find('.js-edit-comment-cancel').on('indico:confirmed', function(evt_) {
```

Comment:
I went for `evt_` since the `evt` from the scope above was somehow shadowing this one.